README.md

```diff
@@ -12,7 +12,7 @@
 
 # Reddit Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_reddit_ads_source/))
 # 📣 What does this dbt package do?
-- Materializes [Reddit Ads staging tables](https://fivetran.github.io/dbt_reddit_ads_source/#!/overview/reddit_ads_source/models/?g_v=1&g_e=seeds) which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/reddit-advertising#schemainformation). These staging tables clean, test, and prepare your reddit_ads data from [Fivetran's connector](https://fivetran.com/docs/applications/reddit-advertising) for analysis by doing the following:
+- Materializes [Reddit Ads staging tables](https://fivetran.github.io/dbt_reddit_ads_source/#!/overview/reddit_ads_source/models/?g_v=1&g_e=seeds) which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/reddit-advertising#schemainformation). These staging tables clean, test, and prepare your reddit_ads data from [Fivetran's connector](https://fivetran.com/docs/applications/reddit-advertising) for analysis by doing the following: 
```

Comment:
All the links in this section are broken.
- The docs link is broken as we do not have the docs included yet in this repo. Once they are added this should resolve itself.
- Both links to Fivetran seem to be broken, I think we need to us the following path instead: `https://fivetran.com/docs/applications/reddit-ads`

Reply:
youre right, links fixed and will regen docs at the end of review

Reply:
@fivetran-reneeli it looks like this link in the updated README points to an empty form of the docs.
![image](https://user-images.githubusercontent.com/74217849/222518245-4b592931-c202-48a4-8b29-efb28ae81845.png)


Would you be able to modify it in order to point to an active page.

Reply:
Hi Joe, are you seeing this when you click on the docs site? I'm not sure what's causing it. I tried regenerating but same outcome. Though when you click into the models themselves the page loads

![image](https://user-images.githubusercontent.com/91097070/223210396-6a689374-5a95-4192-9daa-0b0e19473407.png)

Reply:
I'm not entirely sure what may be causing it 🤔. Would you be able to try and regenerate and see if the same issue persists.

Reply:
Let's try and regen to see if that kick starts things here.

Reply:
Regenerated the docs with newer version of dbt